packages/experimental-utils/src/ast-utils/eslint-utils/ReferenceTracker.ts

```diff
@@ -4,10 +4,12 @@ import * as eslintUtils from 'eslint-utils';
 import { TSESTree } from '../../ts-estree';
 import * as TSESLint from '../../ts-eslint';
 
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 const ReferenceTrackerREAD: unique symbol = eslintUtils.ReferenceTracker.READ;
 const ReferenceTrackerCALL: unique symbol = eslintUtils.ReferenceTracker.CALL;
 const ReferenceTrackerCONSTRUCT: unique symbol =
   eslintUtils.ReferenceTracker.CONSTRUCT;
+/* eslint-enable @typescript-eslint/no-unsafe-assignment */
```

Comment:
we could probably just improve the internal types here?
https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/experimental-utils/typings/eslint-utils.d.ts

Reply:
I went down a bit of a rabbit hole on this one... In theory we could turn them into their own `unique symbol`s, and  use `typeof ...` for the types here.

Unfortunately that then means that the `eslint-utils/ReferenceTracker.d.ts` output file now has an import from `eslint-utils`, which causes [compile complaints](https://github.com/typescript-eslint/typescript-eslint/runs/2596246745):

```
Error: @typescript-eslint/eslint-plugin-internal: ../experimental-utils/dist/ast-utils/eslint-utils/ReferenceTracker.d.ts(1,30): error TS7016: Could not find a declaration file for module 'eslint-utils'. '/home/runner/work/typescript-eslint/typescript-eslint/node_modules/eslint-utils/index.js' implicitly has an 'any' type.
```

At that point I gave up.